docs/rest-server/API.md

```diff
@@ -440,10 +440,16 @@ Status: 200
     createdTime: "createdTimestamp",
     completedTime: "completedTimestamp",
     executionType: "executionType",
-    // Sum of succeededRetriedCount, transientNormalRetriedCount,
-    // transientConflictRetriedCount, nonTransientRetriedCount,
-    // and unKnownRetriedCount
-    retries: retriedCount,
+    // sum of retries
+    retries: retries,
+    retryDetails: {
+      // Failed caused by user's code
+      user: userRetries,
+      // System failed, and it can ensure that it will success within a finite retry times
```

Comment:
"Job failed due to system/plaform error"

Reply:
fixed